#BIS101 F2013 Lecture 2: Mendel

## Notes from last time

Papers are on smartsite or linked.

Github link in announcements.

Reading unit!

ScheduleBuilder takes me to bookstore, has no link to solutions manual. ??

## Incomplete dominance vs. codominance:

blargh locus expresses a protein that makes a horn in the middle of the head.

WT is single horn in middle (draw) H1 allele

Imcomplete dominance: mutant allele H2 fails to produce horn: H1H2 has a small horn

Codominance: mutant allele expressed in different tissues and makes horn on sides of head (draw). H1H2 has three horns (draw). 

H2 vs H3 <strong style="font-size: 150%; color: red;">?</strong> incompletely dominant

## Quiz 1

**Review from lecture 1, dominance (before/as class starts?)**

Cindy crosses an ivory petunia with a red petunia and selfs the F1. The F2 appear as follows: 1/4 red petunias, 1/2 pink petunias, 1/4 ivory petunias. Write out the genotpyes of the F1 and the three different F2.

**Continuation of question 1, dominance**

What is the dominance relationship between the ivory and red allele?
	
## Other forms of inheritance:

#### sex linked

Above we have considered non-sex chromosomes, also called **autosomes**

* instead of heterozygote or homozygote we say heterogametic or homogametic
* works same, but genes on X ≠ Y

Drosophila also have XY system

* Draw Punnet square for w+/w+ Drosophila female (red WT) x w male
* Show color, sex of F1 and F2 products

<img src="/Users/jri/Documents/courses/bis101/griffiths/ch02/figure_02_19.jpg" style="width: 500px;"/>

Reciprocal crosses not same for sex-linked traits

Not all F1 will be the same

Ratios not the same because gene is not present on both chromosomes.          

#### organellar

<strong style="font-size: 150%; color: red;">?</strong> **Eukaryotes** ? defined by presence of nucleus and organelles
Some phenotypes defined by organellar (Mitochondria and chloroplast) genes.

There are some mitochondrial diseases (some deafness, muscle myopathy)

Inheritance: Mito in mammals always maternal, but not in other taxa:

- redwoods both mito and cp are male
- some pine trees, cpDNA male, mtDNA female
- mussels have double uniparental: F mito is maternal, M mito is paternal

**MERFF syndrome pedigree**
* epilepsy  
* short stature  
*  hearing loss  
*  exercise intolerance  
*  poor night vision

<img src="/Users/jri/Documents/courses/bis101/fall2013/images/mito_pedigree.jpg" style="width: 500px;"/>

# Quiz 3

**Lecture 1/Lecture 2 review, sex linkage (couldn't see Jeff's X chromosome notation on the far chalkboard, may need to be updated for consistency)**

Recall that red eye color is dominant to white eye color in fruit flies, and that the eye color locus is on the X chromosome. If you cross a red eyed male (Xw+/Y) to a female heterozygous for the eye color gene (Xw+/Xw), what eye color(s) do you expect in the F1 female flies?

## Mendel's Second Law:

<strong style="font-size: 150%; color: red;">?</strong> Independent assortment 
Unlinked genes assort independently: what happens at one locus, stays at one locus.

<strong style="font-size: 150%; color: red;">?</strong> Cellular basis? Also meiosis:

<img src="/Users/jri/Documents/courses/bis101/fall2014/images/meiosis.jpg" style="width: 500px;"/> but now with 2 chromosomes. Label loci A1, A2.

Work through example. R round > r wrinkled. Y yellow > y green.

<img src="/Users/jri/Documents/courses/bis101/griffiths/ch03/figure_03_04.jpg" style="width: 500px;"/>

* Describe parents phenotype. 
* Ask gametes. 
* Ask F1. 
* Draw Punnett square. 
* Get phenotypes. 
* work out all 16 progeny, figure out phenotype based on dominant/recessive 
* Phenotype ration yellow to green: 3:1 <strong style="font-size: 150%; color: red;">?</strong> for dominance or codom too? called **monohybrid**
* **dihybrid** 9:3:3:1 for phenotype
* for genotype more complex. how many diff genotypes? <strong style="font-size: 150%; color: red;">?</strong> (3^2), most common is RrYy at 4 ways to get it.

Don't worry about Dihbyrid ratios with sex vs. autosomal loci

## Quiz 4

In cats, short hair (S-) is dominant to long hair (ss), white fur color (W-) is dominant to colored fur (ww), and spots (PP) are incompletely dominant to no spots (pp). If you cross a true-breeding white long-haired cat with no spots to a true-breeding heavily-spotted short-haired cat, will the kittens have long or short hair?

<!-- 
## Chi squared

Reality never perfect, e.g. we see 267 vs 93 which we interpret as 3:1 (though not perfect 270:90). But would we interpret 280:80 or 290:70? 

So how to know if close enough? Chi-squared test 

I expect 270:90; i observe 267:93

Chi^2 = Sum((E-O)^2/E); sum over classes. compare to table with df (degrees freedom?)

df = here is number of phenotypic classes -1 (here 2 classes so 1 df)
don't worry about how to calculate df.

sum = 0.13, 1 df, from table 3-1 in book, to be significant at p=0.05 (i.e. <5% chance of observing data under our null model of 3:1) need 3.841.

<strong style="font-size: 150%; color: red;">?</strong> so what do you conclude? (can't reject null hypothesis of expectation, != to saying it is right!), that is 3:1 is a not wrong way of describing this

redo yourself with 244:104. <strong style="font-size: 150%; color: red;">?</strong> If 3:1 is correct, what is predicted numbers? should show the predicted ratio for 3:1 is: 261:87 

value is 4.43 -> reject 3:1 ratio!

Can use for equal segregation (numbers of big A and a in offspring from one locus, association among loci, population genetics, etc.)

will return to Chi-squared for other things in the future. -->
￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼￼
## Predictions: probability

Useful for studying pedigress, for knowing proportions of offspring, gamete production, etc.

**Product rule:** 

For independent events -> multiply

<strong style="font-size: 150%; color: red;">?</strong> Chance of rolling a 2 on a die? 1/6

<strong style="font-size: 150%; color: red;">?</strong> Of rolling a 2 then a 5? 1/6*1/6 = 1/36

* AaBb chance of getting an ab gamete? 1/4! 
* <strong style="font-size: 150%; color: red;">?</strong> Chance to get aBcd from AaBbCcDd -> 1⁄2^4 = 1/16

*Example:*  (save on board)

tb1-M/tb1-T;Wxy/wxy;adh1/adh1; <strong style="font-size: 150%; color: red;">?</strong> Which is dominant? (i didn't say)

<strong style="font-size: 150%; color: red;">?</strong> If I self, what's the chance of getting a tb1-T/tb1-T;wxy/wxy;adh1/ adh1 ?

(1⁄2\*1⁄2)\*(1⁄2\*1⁄2)=1/16=6.25%

<strong style="font-size: 150%; color: red;">?</strong> What's chance of getting tb1-T/tb1-T;wxy/wxy;adh1/Adh1 (0 unless mutation)

**Addition rule:**
For mutually exclusive events (can't both happen)

* you can add
* chance of getting A or a from AaBb (1) chance of getting AB, Ab, or ab <strong style="font-size: 150%; color: red;">?</strong> (0.75)

**Complement rule:**

P(A)=1-P(not A)

So prob. of getting two heads = 1/2 \* 1/2 = 1/4.

Prob. of not getting two heads = 3/4

**Conditional probability**

Pr(A|B) read as A given B = P(A ∩ B) / P(B)  or A intersect B (both A and B happen) divided by prob of B

Example: 

Prob. offspring of monohybrid cross with a>A is Aa given a- phenotype: P(Aa | a-) = P(Aa ∩ a- ) / P(a-) =  (1/2 are both Aa and a-)/(3/4 are a-) = 2/3

* So *given* that individual has the dominant phenotype prob it is het = 2/3

**Hard problem**

How many progeny will i have to grow to get the tb1-T/tb1-T;wxy/wxy;adh1/ adh1 from example above @ 90% certainty?

* Subtract from what's easy to calculate (complement rule). Easy here to say what's prob of never seeing progeny?
* <strong style="font-size: 150%; color: red;">?</strong> What's probability of getting progeny i want in first seed (1/16)
* <strong style="font-size: 150%; color: red;">?</strong> What's probability of not getting progeny i want (15/16) in first seed
* <strong style="font-size: 150%; color: red;">?</strong> What's the probability of getting my progeny if I grow 3 plants?
	- 1-(15/16)^3=17.6% 
* <strong style="font-size: 150%; color: red;">?</strong> what if I grow n plants?
	- 1-(15/16)^n
* So to get 90% certainty, I solve:
	- 1-(15/16)^n=0.9
	- (15/16)^n=0.1 
	- n(ln(15/16))=ln(0.1) 
	- n=ln(0.1)/ln(0.9375)=35.7 == 36 plants

## Quiz 5

What regions of the genome do you miss by only sequencing the exome? Why do the authors argue it is okay for their purposes to miss those regions?
