#BIS101 F2013 Lecture 4: Genetic Mapping 

## Reading for next time:

Include Section 3.4 of ch. 3

## Notes/Questions?

From lecture 2, on conditional probability:  P(A|B)=P(A ∩ B) / P(B)
Example: prob. offspring of monohybrid cross with A>a is Aa given A- phenotype: P(Aa | A-) = P(Aa ∩ A- ) / P(A-) =  (1/2 are Aa and A-)/(3/4 are A-) = 2/3

## RecombinationWhat does independent assortment mean physically <strong style="font-size: 150%; color: red;">?</strong>

Different chromosomes, or far enough apart that recombination breaks up any correlation. "independent" means no correlation.  

Draw AB on two chromosomes. Can you tell me what gamete you get at B locus if I tell you at A locus <strong style="font-size: 150%; color: red;">?</strong>  (NO)

Draw on one chromosome close together.  Now can you tell me <strong style="font-size: 150%; color: red;">?</strong> (yes). These loci are linked.

Mendel got lucky or threw away data maybe. Things close together physically are called **linked**. 

Biologists working on fruit flies began to discover that things weren't always 100% linked or totally independent.  Some genes seemed to move together most but not all the time.  This lead to discovery of recombination, and cytological mechanism of which was finally proven in corn.

## Maps

Chapter illustrates with comparison of london subway maps the idea that there are two kinds of maps of interest to geneticists (define)

**Physical map** <strong style="font-size: 150%; color: red;">?</strong> 

* measured in Kb or kilobase pairs (1000bp) or Mb (megabase 10^6 bp)
        
**genetic map** 

* genetic map is a map of marker or gene order
* <strong style="font-size: 150%; color: red;">?</strong> what's a marker?
* any change in DNA that can be assayed — SNP, CNV, etc. -- and give us genotype.
* can phenotype be used as a marker <strong style="font-size: 150%; color: red;">?</strong> Sure! b/c it can give us a genotype.
* Fig. 4-13 is great stuff.  You might see it again in the future
* not always bigger genetic = bigger physical. why<strong style="font-size: 150%; color: red;">?</strong>
* draw Gore like genetic map
* also hotspots: PRDM9 paper coming later
 
combine the two: if I know genetically my gene is between marker A and B, and I know where A and B are physically -- BAM! I found the piece of DNA where my gene of interest is!

Figure 4.20 does a nice job of showing this.

## Two-point crosses

Let class pick phenotypes dom/recess @ two genes. Use A1/A2 and B1/B2

I want to know whether they are linked. And if so how close?

Two inbred lines A1A1 B1B1 A2A2 B2B2. (save on board)

F1: A1/A2;B1/B2 (save on board)

Test cross <strong style="font-size: 150%; color: red;">?</strong> with F1 (to what?) to A2/A2;B2/B2
 
Why do test cross<strong style="font-size: 150%; color: red;">?</strong> lets me figure out what gametes I got from parent.

With no linkage (draw), Mendel tells me I expect to get <strong style="font-size: 150%; color: red;">?</strong> 1/4 each of (phenotypes):

* A1/A2;B1/B2 (A1-; B1-)
* A1/A2;B2/B2 (A1-; B2)
* A2/A2;B1/B2 (A2; B1-)
* A2/A2;B2/B2 (A2; B2)

What if they're right next to each other on same piece of DNA (draw, ask phenotypes) <strong style="font-size: 150%; color: red;">?</strong> I might see: 1/2 of

* A1/A2;B1/B2 (dominant pheno @ both)
* A2/A2;B2/B2 (recessive pheno @ both)

Linked genes will usually not be so close that recombination impossible.  % recombination tells you how close genes are.  

Let's say instead these two genes are linked. Take my F1 above and testcross. Why testcross and not F1 self<strong style="font-size: 150%; color: red;">?</strong>

Of 2839 offspring I see (phenotypes)

* 1339 (A1-; B1-) --> so what's gamete <strong style="font-size: 150%; color: red;">?</strong> A1;B1
* 1195 (A2A2; B2B2)
* 154 (A1-;B2B2)
* 151 (A2A2;B1-)

In a two point cross, we can see 2 equal classes (no recombination), 4 equal (free recombination, no linkage), or 2 common classes at eq. freq. and 2 uncommon at eq. freq.  

Common classers are **parental** Why <strong style="font-size: 150%; color: red;">?</strong>> he common are (?) no-crossover or parental classes

To calculate recombination we add the recombinant classes and divide by total: 305/2839 = 10.7%

What about if I take a different F1 test cross: 

* 146 (A1-; B1-)
* 157 (A2A2; B2B2)
* 965 (A1-;B2B2)
* 1067 (A2A2;B1-)

Genotype of F1 <strong style="font-size: 150%; color: red;">?</strong> A1_B2/A2_B1 

* explain notation w/ lines; explain **phase**. phenotype and genotype same, but phase differs.
* because we can identify parental types
* What was the genotype of previous F1? A1_B1/A2_B2

We can calculate recombination freq.: 12.9% 

Often want to test: are they linked? Is rec. freq. < 50% ? How<strong style="font-size: 150%; color: red;">?</strong> Chi-sq. test!

Much of this was figured out by Sturtevant as an undergraduate. Instead of doing his homework. Thus my policy of retroactively giving full homework credit to any student who later gets a National Medal of Science or proves their work was the basis of a Nobel prize.

## Three-point testcross

With two markers we can find distance, but with 3 we can do the order of genes.

Add a third phenotype (class<strong style="font-size: 150%; color: red;">?</strong>) 

I make inbred lines, cross A1A1;B1B1;C2C2 (give pheno not geno) w/ A2A2;B2B2;C1C1
    
I know genotype of F1: A1_B1_C2;A2_B2_C1 but I don't know order! Could be A1_C2_B1 !

Test cross:

* A1C2B1 580 (showing gametes for ease, order listed here or above is NOT informative)
* A2C1B2 592 
* A1C2B2 45
* A2C1B1 40 
* A2C2B2 89
* A1C1B1 94 
* A2C2B1 3
* A1C1B2 5

Can identify parentals as most common (A1C2B1, A2C1B2)

Can identify double DCO (what are these<strong style="font-size: 150%; color: red;">?</strong>) as least (A2C2B1, A1C1B2). 

Why are these the least <strong style="font-size: 150%; color: red;">?</strong>

DCO differ by one from parent (draw)

ID the DCO and you know order!  

* F1s were C2_A1_B1 / C1_A2_B2

Now distances:

First let's ignore B.  We can then treat C2 and A1 as a two-point cross (annotate rec. and parental). 13.2%

Now ignore C and do A and B -> 6.4 % 

The other way to find order is to do all 3 pairwise distances: the biggest distance are the outside two!

In a simple world, we'd be done & freq. of double crossovers would be <strong style="font-size: 150%; color: red;">?</strong> 

* 0.064 * 0.132 = 0.00845, and of 1448 gametes we expect ~12 double crossover gametes
* We see 8?  what the heck?

Crossing-over takes physical space, and you can't have two close to each other.  so having one here makes the chance of having another lower

We calculate this decrease as interference. simply 1-obs/exp, or in this case 1-8/12 = 33%

### Linkage mapping

higher % recombination = bigger distance apart (in general)

* can use this plus order from 3-point crosses to build genetic maps of where genes are in relation to each other

For short distances, RF% interpreted as crossovers. Measured in cM (after Thomas Hunt Morgan, Sturtevant's boss).

So 3cM means 3% chance of a crossover.

For longer distances, not all crossovers will be observed (DCO in example above). Mathematical functions desc. in book have been devised to turn recombination frequency into number of crossovers.

Note maximum RF% is always 50%! Why can never get 100% <strong style="font-size: 150%; color: red;">?</strong>

* Think about separate chromosomes
* Check out fig. 4-19 in book showing how even for DCO on avg. get 50%

Haldane's mapping fxn. Relates ovsered RF with expected # of CO and then to what RF *should* be given that # of CO.

* RF=0.5*(1-e^-m) 
* where m is mean number crossovers in interval. 
* So for RF of 27.5% we get m=0.8
* Since 1 CO = 50% RF = 50cM, to turn this into cM we multiply by 50 = 40 cM (note bigger than 27.5!!)


## Mechanism of crossing over

Come back to hopefully.  Don't memorize. Draw with color.

Happens in chromatids. in tetrad of sister chromatids.  So when<strong style="font-size: 150%; color: red;">?</strong> Meiosis!

Involves double-strand breaks in DNA. Fig. 4-21 in book.

<img src="/Users/jri/src/bis101/griffiths/ch04/figure_04_21.jpg" style="width: 500px;"/>

Strand invasion, extension, formation of Holliday junction

Whole shebang can be resolved to form: no crossover, crossover

Leaves heteroduplex DNA which can then be resolved as **gene conversion** or not.

Draw simple gene conversion:

* ABC
* ABC
* abc
* abc

into

* ABC
* ABC
* aBc
* aBc

Different from DCO because now ALL gametes are B!

