# BIS101 F2013 Lecture 14: Proteins


DNA -> transcription -> RNA -> translation -> protein

## Protein

A chain of amino acids. Called a **polypeptide** (DRAW)

<img src="http://upload.wikimedia.org/wikipedia/commons/thumb/c/ce/AminoAcidball.svg/702px-AminoAcidball.svg.png" style="width: 200px;"/>

Has an amino end and a carboxyl end. Polypeptide/protein is formed by bonding of amino acids. (show). Lose one H2O molecule and make covalent bond.

Protein has an N and C terminus of a protein, convention is to write N->C direction but this is not fundamental directionality same way 5' to 3' is.

Can one gene make more than one protein <strong style="font-size: 150%; color: red;">?</strong> (yes, alternative splicing). These are called **isoforms**.

Parts of a sequence that have a particular function (e.g. DNA-binding) are referred to as a **domain** 

How do proteins know where to go? Often include a short bit of AA at beginning that functions as a **signal sequence**. This bit removed when protein arrives to right part of cell and is further processed.ÃŸ

Combo of all proteins in the cell <strong style="font-size: 150%; color: red;">?</strong> The proteome.

## Genetic code

Already discussed how DNA/RNA come in triplets called codons that code for amino acids.  Show top left of Figure. Leucene vs Phen. for 3rd position. Leucene for 3rd position  and Leucine vs. Proline for 2nd position. Synonymous vs. non.

<img src="/Users/jri/src/bis101/griffiths/ch09/figure_09_05.jpg" style="width: 500px;"/>

These are recognized by a special RNA called a transfer RNA

<img src="/Users/jri/src/bis101/griffiths/ch09/figure_09_06a.png" style="width: 400px;"/>

Two active parts are AA attachment site and **anticodon** loop.  If mRNA has GCA what will anticodon be <strong style="font-size: 150%; color: red;">?</strong> CGU

How many different AA are there <strong style="font-size: 150%; color: red;">?</strong> 20

How many different codons are there <strong style="font-size: 150%; color: red;">?</strong> 64

But organisms don't need to have 61 tRNAs (why 61 and not 64<strong style="font-size: 150%; color: red;">?</strong> because stop codons which are recognized by special proteins and not tRNA)

So how do we have <61 tRNAs recognize 61 codons?  Wobble rule. Codon-anticodon pairing is a tad more relaxed than normal DNA-DNA or DNA-RNA pairing.

<img src="/Users/jri/src/bis101/griffiths/ch09/table_09_01.jpg" style="width: 300px;"/>

## Ribosomes

Molecular machines that make protein from RNA. Comprised of small subunit, large subunit, and each is a complex of large number of ribosomal proteins and several ribosomal RNA. 

<img src="/Users/jri/src/bis101/griffiths/ch09/figure_09_12b.jpg" style="width: 500px;"/>

DRAW. label LG and SM subunits. label EPA (will come back to). label tunnel, draw polypeptide chain. label decoding center and peptidyltransferase center. show mRNA 5' and 3'.

Some antibiotics work by binding to specific regions of ribosomal RNA and blocking translation (e.g. exit tunnel). Some antibiotic resistance is e.g. mutation making a bigger exit tunnel!

## Initiation, elongation, termination

#### Initiation

We know first codon of the protein is <strong style="font-size: 150%; color: red;">?</strong> AUG which is <strong style="font-size: 150%; color: red;">?</strong> methionine. But how ribosome knows where to start?

Prokaryotes have **Shine-Delgarno** sequence (often AGGAGGU) which binds small subunit and proteins called **initiation factors**, grabs the intitiation tRNA (different from standard methionine tRNA) then binds large subunit and starts translating.

* In prokaryotes where does this happen <strong style="font-size: 150%; color: red;">?</strong> What does that mean about when this can happen <strong style="font-size: 150%; color: red;">?</strong> During transcription!
* In Kudla, why didn't they see effect of variability in S-D sequence on total protein levels? (because they only modified synonymous sites in the coding region!)

In Eukaryotes initiation factors and small subunit recognize the 5' cap placed on mature mRNAs and scans for AUG, joins with large subunit

#### Elongation

Ribosome moves 5'->3' along mRNA. 

what does each center do <strong style="font-size: 150%; color: red;">?</strong>

A (aminoacyl) site binds incoming tRNA w/ matching codon

As ribosome moves, polypeptid is transferred to tRNA in A which moves to P (peptidyl) site. Old tRNA from P site moved into E (Exit) site.

What about introns <strong style="font-size: 150%; color: red;">?</strong> they are already spliced out, we don't have to worry about them!

#### Termination

We get to a stop codon. Which are <strong style="font-size: 150%; color: red;">?</strong> UAG UAA UGA. Recognized by a protein called a release factor that binds to stop codon.

## Post-translation

So after protein is made, then what <strong style="font-size: 150%; color: red;">?</strong>

#### Shape

For proteins shape matters -- how it's folded often determins what it does. e.g. forming the active site of an enzyme.

Protins have several levels of structure:

* **Primary** structure: sequence of the protein
* **Secondardy** strcuture: folding of the polypeptide. Common ones include a alpha-helix, and a pleated sheet (like a folding screen)
* **Tertiary** structure: folding of secondary structures
* **Quaternary** structure: combination of multiple subunits. e.g. hemoglobin is multiple subunits joined by weak bonds. **Dimer** if two subunits. **Homodimer** vs. **heterodimer**.

Protein folding undertaken by large proteins called chaperones.

Predicting how proteins fold and what determines when the misfold is hugely important and really really hard to do. You can help! 267,000 home computers doing this: 

folding.stanford.edu cluster computing

fold.it video game

#### Modifications

Phosphorylation: can change shape because phosphate groups negatively charged. (serine, threonine, tyrosine). Done by kinases. Ev. of importance is there are > 1000 kinases in many organisms!

We already saw acetylation and methylation of lysine in histone proteins.

Ubiquitination in Eukaryotes another common modification, which adds a large number of ubiquitin  side chains and marks a protein for degradation.